<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetSunSetRise</class>
 <widget class="QWidget" name="GetSunSetRise">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>284</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sunset Sunrise</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Longitude</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Latitude</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QSplitter" name="splitter_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLineEdit" name="leLongtitude">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="leLatitude">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QRadioButton" name="rbEast">
      <property name="text">
       <string>East meridian</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbWest">
      <property name="text">
       <string>West meridian</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QRadioButton" name="rbSunSet">
      <property name="text">
       <string>Sunset</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbSunRise">
      <property name="text">
       <string>Sunrise</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Zenith of the sun</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Timezone</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QComboBox" name="cbTimeZone">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QComboBox" name="cbSunZenith">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>OFFICIAL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CIVILIAN</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NAVIGATIONAL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ASTRONOMICAL</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QPushButton" name="bCalc">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QLabel" name="lResults">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>None</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
